packages/docs/gatsby-config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
                 spaceId: '2rb53inahems',
                 accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
                 downloadLocal: true,
-                environment: process.env.NODE_ENV === 'development' ? 'dev' : 'master',
+                environment: process.env.NODE_ENV === 'development' ? 'page-updates' : 'master',
```

Comment:
What's going to happen as soon as this is merged into master the deploy will start to kick off. We will end up seeing the JSON in the browser (like we are in the preview). 

1. Here we change the `master` to also point to `page-updates` contentful branch `environment: process.env.NODE_ENV === 'development' ? 'page-updates' : 'page-updates',`
This means when it's deployed (both preview and prod) we will see the new pages and original content. 

2. After production deploy; in contentful we then make `page-updates` the "master" branch (but don't delete "page-updates" just yet)
3. We then make another PR here to change this to look like this does currently `environment: process.env.NODE_ENV === 'development' ? 'page-updates' : 'master',`


This should avoid any downtime/JSON in the prod site. Does that make sense?

Reply:
Perfect. Great idea! I'll get a quick update pushed for this. Thanks Cameron